app/src/components/TxSummary.tsx

```diff
@@ -96,8 +101,18 @@ const TxSummary: FC<TxSummaryProps> = ({
               <div className="flex">
                 <div className="font-bold">Duration</div>
               </div>
-              <div className="items-right flex">
+              <div className="items-right flex items-center space-x-0.5">
                 <div className="text-turtle-foreground">{durationEstimate}</div>
+                {direction === Direction.WithinPolkadot && (
+                  <Tooltip
+                    showIcon={false}
+                    content={
+                      'Transfer should be done in ~30s but our service takes up to 2 minutes to catch up'
+                    }
+                  >
+                    <Info className="h-3 w-3 text-turtle-level6" />
```

Comment:
Let's leave it as you initially did:

```suggestion
                    <Info className="h-3 w-3 text-turtle-foreground" />
```